ScribusGeneratorBackend.py

```diff
@@ -214,8 +214,9 @@ def parse_data(self):
         if extension == '.csv':
             # .. from CSV file
             data = self.load_csv(data_file)
+            logging.debug(data)
 
-            if len(data) <= 1:
```

Comment:
why this change ? if data.len is 1 then there's only titles, no values to substitute > skip.

Reply:
The variable 'data' is 'OrederdDict' list types which has title and data row pair.
I was faced getting this error when loading csv file which has only one record.